CMakeLists.txt

```diff
@@ -88,15 +87,21 @@ if (CMAKE_COMPILER_IS_GNUCC)
 	if(DEFINED ENV{TRAVIS} AND DEFINED ENV{CI})
 		set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Werror")
 	endif()
-endif()
 
-if(APPLE)
-	#full Single Unix Standard v3 (SUSv3) conformance (the Unix API)
-	add_definitions(-D_DARWIN_C_SOURCE)
+elseif (CMAKE_C_COMPILER_ID STREQUAL "Clang")
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -pedantic -Wno-unused-parameter -Wno-sign-compare")
+	if(APPLE)
```

Comment:
If someone tries to use gcc on Apple, this won't work. Not sure if people do that. Maybe it's ok this way.

Reply:
OK will fix.